resources/zib-HealthProfessional-PractitionerRole.xml

```diff
@@ -0,0 +1,153 @@
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-healthprofessional-v3.5-2020EN" />
+    <uri value="https://zibs.nl/wiki/HealthProfessional-v3.5(2020EN)" />
+    <name value="Zib HealthProfessional-v3.5(2020EN)" />
+  </mapping>
+  <mapping>
+    <identity value="zib-contactinformation-v1.2-2020EN" />
+    <uri value="https://zibs.nl/wiki/ContactInformation-v1.2(2020EN)" />
+    <name value="zib ContactInformation-v1.2(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="PractitionerRole" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/PractitionerRole" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="PractitionerRole">
+      <path value="PractitionerRole" />
+      <comment value="The zib HealthProfessional is mapped for all but one concept (HealthProfessionalRole) to Practitioner and PractitioneRole. PractitionerRole covers the recording of the location and types of services that HealthProfessionals are able to provide for a HealthcareProvider. The zib concepts 'HealthProfessionalIdentificationNumber', 'Specialty' and 'HealthcareProvider' are therefore mapped to PractitionerRole. &#xD;&#xA;&#xD;&#xA;Although the zib concept name HealthProfessionalRole feels a match for PractitionerRole, this is not the case based on the definition and values of that concept. In contrary to the zib, the more clinical resources contain the role information the Healthprofessinal fullfills in the that specific case. An example is the `Procedure.performer.role` element. This prevents creating many HealthProfessional instances." />
+    </element>
+    <element id="PractitionerRole.identifier">
+      <path value="PractitionerRole.identifier" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="PractitionerRole.identifier:BIG">
+      <path value="PractitionerRole.identifier" />
+      <sliceName value="BIG" />
+      <patternIdentifier>
+        <system value="http://fhir.nl/fhir/NamingSystem/big" />
+      </patternIdentifier>
+    </element>
+    <element id="PractitionerRole.identifier:UZI">
+      <path value="PractitionerRole.identifier" />
+      <sliceName value="UZI" />
+      <patternIdentifier>
+        <system value="http://fhir.nl/fhir/NamingSystem/uzi-nr-pers" />
+      </patternIdentifier>
+    </element>
+    <element id="PractitionerRole.identifier:AGB">
+      <path value="PractitionerRole.identifier" />
+      <sliceName value="AGB" />
+      <patternIdentifier>
+        <system value="http://fhir.nl/fhir/NamingSystem/agb-z" />
+      </patternIdentifier>
+    </element>
+    <element id="PractitionerRole.practitioner">

```

Comment:
Moet dit element niet 1..1 worden? De zib veronderstelt dat het specialisme onderdeel is van de zorgverlener; een specialisme kan volgens de zibs dus niet los bestaan.

Reply:
Nee ik denk het niet. Volgens de zib kan er alleen een specialisme worden doorgegeven (alles is optioneel). Als je de Practitioner verwijzing verplicht stelt zeg je in feite dat ook minimaal één van de concepten die gemapt zijn in Practitioner aanwezig moet zijn. Een lege resource kan immers niet. Dat gaat er dus misschien wat over in de zib- laag?

Reply:
Fair enough